src/sentry/models/groupassignee.py

```diff
@@ -34,6 +35,37 @@
 
 
 class GroupAssigneeManager(BaseManager):
+    def get_assigned_to_data(
+        self, assigned_to: Team | RpcUser, assignee_type: str, extra: Dict[str, str] | None = None
+    ) -> Dict[str, str | Any | None]:
+        data = {
+            "assignee": str(assigned_to.id),
+            "assigneeEmail": getattr(assigned_to, "email", None),
+            "assigneeType": assignee_type,
+        }
+        if extra:
+            data.update(extra)
+
+        return data
+
+    def get_assignee_data(self, assigned_to: Team | RpcUser) -> tuple[str, str, str] | None:
```

Comment:
> If someone can help with that last typing error that'd be great cause I tried everything I can think of.

It looks like the error is because `get_assignee_data()` says it can return None which would fail since you're unpacking the tuple. But it looks like that's not right so you can just remove the None: 

```suggestion
    def get_assignee_data(self, assigned_to: Team | RpcUser) -> tuple[str, str, str]:
```

Reply:
Oh I put the or none thing since it could just raise an error which I guess isn't returning none but more of a lack of a return